src/compiler/legacy/get_err_msg.js

```diff
@@ -0,0 +1,335 @@
+var util     = require('util'),
+    ErrCodes = require('./err_codes');
+
+module.exports = function (err) {
+    var msg = 'UNKNOWN_ERROR',
+        m   = function () {
+            msg = util.format.apply(this, arguments);
+        };
+
+    switch (err.type) {
+        //Fixture file
+        case ErrCodes.FIXTURE_DIRECTIVE_REDEFINITION:
+            m(
+                '(line %s): @fixture directive redefinition. @fixture directive should be used once per test fixture file.',
```

Comment:
Can a directive be defined or redefined? It's not a variable or an identifier so... I guess no.

```
                '(line %s): @fixture duplicated. The @fixture directive can only be used once per test fixture file.',
```